### DISCUSSION TITLE
https://haipingblog.vercel.app/%E7%AC%94%E8%AE%B0/%E6%95%B0%E5%AD%A6%E7%AC%94%E8%AE%B0/%E6%95%B0%E5%AD%A6%E7%AB%9E%E8%B5%9B2.html

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/nolebase-template#18] [Announcements]
# https://haipingblog.vercel.app/%E7%AC%94%E8%AE%B0/%E6%95%B0%E5%AD%A6%E7%AC%94%E8%AE%B0/%E6%95%B0%E5%AD%A6%E7%AB%9E%E8%B5%9B2.html

海上生明月，天涯共此时

https://haipingblog.vercel.app/%E7%AC%94%E8%AE%B0/%E6%95%B0%E5%AD%A6%E7%AC%94%E8%AE%B0/%E6%95%B0%E5%AD%A6%E7%AB%9E%E8%B5%9B2.html

**Comment by OnHaiping** [comment]
这是一个测试评论
